index.html

```diff
@@ -1 +1,26 @@
-
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width,initial-scale=1" />
```

Comment:
What is this?

Reply:
That makes it so that "50%" for example is 50% for whatever device not just a chromebook.

Reply:
???